docs/genome_to_collection.md

```diff
@@ -0,0 +1,235 @@
+gfu_service_ver=dev
+cbs_max_tasks=10
+
+PYTHONPATH=. python src/loaders/workspace_uploader/workspace_uploader.py \
+    --workspace_id $workspace_id \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_verion \
+    --env $env \
+    --token_filepath $token_filepath \
+    --au_service_ver $au_service_ver \
+    --gfu_service_ver $gfu_service_ver \
+    --cbs_max_tasks $cbs_max_tasks \
+    --load_id $load_id  
+```
+
+# Step 3: Execute tools on FASTA files
+
+Once the source data files are prepared, execute tools on these FASTA files. You can choose from various available 
+tools such as gtdb_tk, checkm2, microtrait, or mash.
+
+```commandline
+# update arguments as needed
+tool=gtdb_tk  # available tools: gtdb_tk, checkm2, microtrait, mash
+load_ver=$source_verion
+source_file_ext=.fa
+
+PYTHONPATH=. python src/loaders/jobs/taskfarmer/task_generator.py \
+    --tool $tool \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_ver \
+    --load_ver $load_ver \
+    --source_file_ext $source_file_ext \
+    --submit_job
+```
+
+# Step 4: Parse and Load Tool Outputs
+
+## Parse tool results
+
+### Parse tool computation results
+Parsing tool results for extracting insights from the processed data. This step involves parsing tool computation 
+results and genome taxa count information.
+
+```commandline
+# Parse tool computation results
+PYTHONPATH=. python src/loaders/genome_collection/parse_tool_results.py \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_ver \
+    --load_ver $load_ver \
+    --env $env
+	
+# Parse genome taxa count information
+attri_file=${kbase_collection}_${load_ver}_checkm2_gtdb_tk_kbcoll_genome_attribs.jsonl
+PYTHONPATH=. python src/loaders/genome_collection/compute_genome_taxa_count.py \
+    ../import_files/$env/$kbase_collection/$load_ver/$attri_file \
+    --load_ver $load_ver \
+    --kbase_collection $kbase_collection \
+    --env $env \
+    --input_source genome_attributes
+```
+
+## Load parsed results to ArangoDB
+
+```commandline
+# set up an SSH tunnel (Not required when using an internal KBase machine such as dev03) 
+USER_NAME=user_name                 # user name for login1.berkeley.kbase.us
+FORWARD=localhost:48000
+ssh -f -N -L $FORWARD:10.58.1.211:8531 \
+ $USER_NAME@login1.berkeley.kbase.us
+
+# execute arangoimport (Please ask system admin for the arangoDB credentials)
+PARSED_FILE=json_file_path          # the file path generated by the parsing script
+ARANGO_USER=arango_username         # arangoDB user name
+ARANGO_PW=arango_password           # arangoDB password
+ARANGO_DB=collections_dev           # arangoDB database name
+ARANGO_COLL=kbcoll_genome_attribs   # arangoDB collection name
+
+arangoimport --file $PARSED_FILE \
+    --server.endpoint tcp://$FORWARD \
+    --server.username $ARANGO_USER \
+    --server.password $ARANGO_PW \
+    --server.database $ARANGO_DB \
+    --collection $ARANGO_COLL \
+    --on-duplicate update
+```
+
```

Comment:
I think instructions for the homology files are missing

Reply:
I actually don't know how homology works. Do you mind adding that section after this PR?

Reply:
I think you just need to say something along the lines of "for these files, upload them to the homology server using the instructions here https://github.com/jgi-kbase/AssemblyHomologyService and use the namespace when creating the collection with a homology matcher." 

You'll need to pretty that up a bit though, that's just the general idea

Reply:
👍

Reply:
Can you denote which files are the homology files?

Reply:
👍 added the sketch file name.

Reply:
There's 2 files that are product for the homology service, the sketch file and the sequences file

Reply:
I am guessing you are referring to the sequence metadata file. I included that file.

Reply:
:+1: